catalog/ui/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -161,6 +214,22 @@ define([
                 if (!(result.north !== undefined && result.south !== undefined && result.west !== undefined && result.east !== undefined) && this.get('usngbb')) {
                     result = converter.USNGtoLL(this.get('usngbb'));
                 }
+                if (!(result.north !== undefined && result.south !== undefined && result.west !== undefined && result.east !== undefined) && this.isUtmUpperLeftDefined()) {
+                    var utmParts = this.parseUtmUpperLeft();
+                    if (utmParts !== undefined) {
```

Comment:
nit : there may be some potential to consolidate this code into one function and use it for upper left and lower right